Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'sass-rails'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 
+gem 'webpack-rails'
```

Comment:
Any reason why you would use this over [`webpacker`](https://github.com/rails/webpacker)? I'm not sure which is 'better' though >_<